docs/concepts/account.md

```diff
@@ -106,7 +106,7 @@ See our [action specifications](https://nomicon.io/RuntimeSpec/Actions.html) sec
 
 A `FunctionCall` key is unique as it only has permission to call a smart contract's method(s) that _do not_ attach Ⓝ as a deposit (i.e. payable functions). These keys have the following three attributes:
 
-1) `allowance` - amount of Ⓝ loaded onto the key to pay for gas fees  _(0.25 default)_
+1) `allowance` - the amount of Ⓝ the key is allowed to spend on gas fees _(required)_
```

Comment:
```suggestion
1) `allowance` - the amount of Ⓝ the key is allowed to spend on gas fees _(optional - default: `null`)_
```

As you stated, this field isn't actually required. We should note the default value of `null` for now until this is updated with a default value.